cricPredict/Contest/models.py

```diff
@@ -5,6 +5,7 @@
 
 # Create your models here.
 class League(models.Model):
+    api_id = models.IntegerField(default=0)
```

Comment:
we shouldn't be saving API Ids in the database, you can specify it as an environment variable instead